### DISCUSSION TITLE
get_model_feature_importance() is empty

### DISCUSSION THREAD

**Discussion by rcoughlan-1980** [autogluon/autogluon#1315] [General]
Hi All,

I have made some great models and I want to look at the feature importances (calculated from training) of the individual models in my predicator.

I called 
specific_model = predictor._trainer.load_model(model_to_use)
specific_model.get_model_feature_importance()

but all it returns is 

{}

Does anyone have any ideas...I remade the models and the problem persisted. Is there something wrong with my code. Thanking you in advance!!

Ruth

**Comment by Innixma** [comment]
Hi Ruth,

It is recommended to not load individual models like this (thats why `_trainer` is private).

Instead, you can just do:

```
predictor.feature_importance(test_data, model=model_to_use)
```
